pre-commit.sh

```diff
@@ -14,7 +14,7 @@ gitleaksEnabled=$(git config --bool hooks.gitleaks)
 # but you're actually trying to commit:
 #   database-pass: a-real-damn-password
 # then, you need to see the full output to realize your mistake
-cmd="$HOME/bin/gitleaks --unstaged --verbose --leaks-exit-code=1 --config-path=$HOME/.git-support/gitleaks.toml"
+cmd="$HOME/bin/gitleaks protect --staged --config=$HOME/.git-support/gitleaks.toml --verbose"
```

Comment:
This is the major change necessary to support people who already had installed `gitleaks` v8 via Homebrew. See https://github.com/cloud-gov/caulking/pull/70. 

We also need the `--staged` flag to ensure that staged changes are scanned. It seems that even with `--staged`, unstaged changes are scanned too, which I added a unit test to prove